### DISCUSSION TITLE
SC4S Juniper (junos) firewall line breaking issue

### DISCUSSION THREAD

**Discussion by rashishsharma** [splunk/splunk-connect-for-syslog#1823] [Q&A]
Hi All, 

We are integrating juniper firewalls using SC4S to Splunk cloud. We have tried integrating the firewall over TLS and TCP. using both the methods we see line breaking issue.

My container version is 2.34.2

Please suggest

Regards,

Rashish Sharma

**Comment by rjha-splunk** [comment]
RFC3164 and 5424 
without ietf frame: use 514 for tcp udp and 6514 for tls
with IETF Frame: Use 601 with tcp/udp and 5425 for tls.

What port you are sending the data , can you DM( by external slack) the pcap file.
